pootle/static/js/editor/app.js

```diff
@@ -2209,26 +2247,45 @@ PTL.editor = {
 
 
   /* Accepts a suggestion */
-  acceptSuggestion(suggId, { skipToNext = false } = {}) {
-    UnitAPI.acceptSuggestion(this.units.getCurrent().id, suggId)
+  handleAcceptSuggestion(
+    suggId, { requestData = {}, isSuggestionChanged = false } = {}
+  ) {
+    PTL.editor.acceptSuggestion(suggId, { requestData, isSuggestionChanged });
+  },
+
+  acceptSuggestion(
+    suggId, { requestData = {}, skipToNext = false, isSuggestionChanged = false } = {}
+  ) {
+    if (!isSuggestionChanged) {
```

Comment:
The problem with this change is, `acceptSuggestion` no longer accepts a suggestion, but it does more stuff. OTOH `handleAcceptSuggestion` is specifically crafted to deal with the feedback form, therefore this is where this logic belongs to. This would be even more obvious to see if all this logic lived in a container/controller component (which by the way, would be a very good idea to do in order to reduce coupling).